examples/axp_optimizer_example.py

```diff
@@ -0,0 +1,143 @@
+        #             "riskmodeldate": "09/01/2020",
+        #             "backtestdate": "09/01/2020",
+        #             "cashflow": "0"
+        #         },
+        #         "outputtypes": {
+        #             "trades": {
+        #                 "identifiertype": "SedolChk",
+        #                 "includecash": false
+        #             }
+        #         }
+        #     }
+        # }
+        axp_optimizer_strategy = AxiomaEquityOptimizerStrategy(
+            id="Client:/Optimizer/CN_TEST")
+        axp_optimizer_account = OptimizerAccount(
+            id="CLIENT:/OPTIMIZER/IBM.ACCT")
+        axp_optimizer_optimization = Optimization(
+            risk_model_date="09/01/2020",
+            backtest_date="09/01/2020",
+            cashflow="0"
+        )
+        axp_optimizer_trades_list = OptimizerTradesList(
+            identifier_type="SedolChk", include_cash=False)
+        axp_optimizer_output_types = OptimizerOutputTypes(
+            trades=axp_optimizer_trades_list)
+        axp_optimizer_parameters = AxiomaEquityOptimizationParameters(
+            strategy=axp_optimizer_strategy,
+            output_types=axp_optimizer_output_types,
+            account=axp_optimizer_account,
+            optimization=axp_optimizer_optimization
+        )
+        axp_optimization_parameters_root = AxiomaEquityOptimizationParametersRoot(
+            data=axp_optimizer_parameters)
+
+        axp_optimizations_api = AXPOptimizerApi(api_client)
+
+        post_and_optimize_response = axp_optimizations_api.post_and_optimize(
+            axioma_equity_optimization_parameters_root=axp_optimization_parameters_root
+        )
+
+        if post_and_optimize_response[1] == 201:
+            output_optimization_result(post_and_optimize_response[0]['data'])
+        else:
+            optimization_id = post_and_optimize_response[2]["X-Factset-Api-Calculation-Id"]
+            print("Calculation Id: " + optimization_id)
+
+            status_response = axp_optimizations_api.get_optimization_status_by_id(id=optimization_id)
+
+            while status_response[1] == 202:
+                max_age = '5'
+                age_value = status_response[2].get("cache-control")
+                if age_value is not None:
+                    max_age = age_value.replace("max-age=", "")
+                print('Sleeping: ' + max_age)
+                time.sleep(int(max_age))
+                status_response = axp_optimizations_api.get_optimization_status_by_id(optimization_id)
+
+            if status_response[1] == 201:
+                print("Optimization Id: " + optimization_id + " Succeeded!!!")
+                result_response = axp_optimizations_api.get_optimization_result(id=optimization_id)
+                output_optimization_result(result_response[0]['data'])
+            else:
+                print("Optimization Id:" + optimization_id + " Failed!!!")
+                print("Error message : " + status_response[0].error)
```

Comment:
I think this needs to be `status_response[0].errors`. You will have to make the change in other locations as well